demo/index.html

```diff
@@ -24,6 +24,12 @@
             script.src = '../../webrtc-adapter/release/adapter.js';
             script.onload = resolve;
             document.body.appendChild(script);
+
+            // npm version has different dir structure compared to bower
+            var script2 = document.createElement('script');
+            script2.src = '../../webrtc-adapter/out/adapter.js';
```

Comment:
Maybe we can do the NPM one and then try the other one if that fails (e.g., in an `onerror` handler)?